File: src/EducationSystem/EducationSystem.WebApp/App/src/models/Answer.ts
import Model from './Model'
import {AnswerStatus} from '../common/enums'

export default class Answer extends Model {
  public Text: string = ''
  public IsRight?: boolean
  public QuestionId?: number
  public Status?: AnswerStatus
  
  constructor(text: string, isRight?: boolean) {
    super()
    
    this.Text = text
    this.IsRight = isRight
  }
}

File: src/EducationSystem/EducationSystem.Models/Rest/Answer.cs
using EducationSystem.Enums;
using EducationSystem.Models.Rest.Basics;

namespace EducationSystem.Models.Rest
{
    public class Answer : Model
    {
        public string Text { get; set; }

        public bool? IsRight { get; set; }

        public int? QuestionId { get; set; }

        public AnswerStatus? Status { get; set; }
    }
}